### PR TITLE
Minor improve docs example

### PR DESCRIPTION
**Pull request by shaopeng-gh**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

- **What is the current behavior?** (You can also link to an open issue here)
running in Windows will have `\` and we should replace with `/`.

- **What is the new behavior (if this is a feature change)?**
improve the example

- **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,8 +121,8 @@ for (const issue of issues) { // issues from your linter in any format
         messageText: err.msg,                                     // Ex: "any is forbidden !"
         ruleId: err.rule,                                         // Ex: "no-any"
         fileUri: process.env.SARIF_URI_ABSOLUTE                   // Ex: src/myfile.ts
-            ? "file:///" + fileNm.replace(/\\/g, "/")
-            : path.relative(process.cwd(), fileNm)
+            ? pathToFileURL(fileNm)
```
Comment (shaopeng-gh) [inline README.md:124]: >pathToFileURL(fileNm) [](http://example.com/codeflow?start=14&length=21)

let me know if you think this looks better with same function

File: README.md [hunk 2]
```diff
@@ -119,8 +121,8 @@ for (const issue of issues) { // issues from your linter in any format
...
+            : path.relative(process.cwd(), fileNm).replace(/\\/g, '/'),
```
Comment (shaopeng-gh) [inline README.md:125]: >.replace(/\\/g, '/') [](http://example.com/codeflow?start=50&length=20)

running in windows will have `\`, in the relative path we should have `/`

**Comment by shaopeng-gh** [comment]
@nvuillam please review.

**Review by nvuillam — APPROVED** [review]
Seems legit, thanks for your contribution :)

**Pull request merged**